src/models/searchservice/response/DirectAnswer.ts

```diff
@@ -1,23 +1,61 @@
 import { Result } from './Result';
+import { DirectAnswerType } from './DirectAnswerType';
+import { Snippet } from './Snippet';
 
 /**
  * A direct answer to a search.
  *
  * @public
  */
 export interface DirectAnswer {
+  /** The {@link DirectAnswerType}. */
+  type: DirectAnswerType | string;
   /** The result of the direct answer. */
   value: string;
   /** The entity associated with the direct answer. */
   relatedResult: Result;
   /** The vertical key of the direct answer. */
   verticalKey: string;
-  /** The name of the entity of the direct answer. */
+}
+
+/**
+ * A direct answer which was found within a document.
+ *
+ * @public
+ */
+export interface FeaturedSnippetDirectAnswer extends DirectAnswer {
+  /** {@link DirectAnswerType}.FeaturedSnippet. */
+  type: 'FEATURED_SNIPPET';
```

Comment:
It's possible to use the enum directly here (e.g. DirectAnswerType.FeaturedSnippet), however that causes the documentation generation to creates a link to a page that doesn't exist. This is a small workaround which allows the enum to be used for checking the type without causing  a broken link to be generated.

Reply:
Is this due to a known bug with TSDoc?

Reply:
It's actually an issue with api-extractor. I'll message the devs and see if it's a bug

Reply:
Do we want to form our code around documentation tools? I would expect this to be only of type `DirectAnswerType`, as it's easier to reason what values can go into the type

Reply:
I wanted to avoid having broken links in our documentation, however it might be fine if we put a working link right next to it